src/components/header/Header.style.tsx

```diff
@@ -84,3 +92,60 @@ export const TabLink = styled(Link)`
     font-size: 18px;
   }
 `
+
+export const HeaderDrawer = styled(Drawer)`
+  ${media.lg`
+      display: none;
+    `}
+
+  .ant-drawer-body {
+    margin: 4px 4px 0px 4px;
+    padding: 0;
+    border-radius: 5px;
+  }
+
+  .ant-drawer-content {
+    background-color: transparent;
+  }
+
+  /* Make sure following id's are defined in svg */
+  #swap_icon,
+  #stake_icon,
+  #wallet_icon {
+    & > * {
+      fill: ${palette('text', 1)};
+    }
+  }
+`
+
+export const HeaderDrawerItem = styled(Row)`
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-color: ${palette('background', 2)};
+  background-color: ${palette('background', 0)};
+  align-items: center;
+  transition: none;
+  height: 60px;
+  display: flex;
+  text-transform: uppercase;
+  font-family: 'MainFontSemiBold';
+  font-size: 18px;
+  color: ${palette('text', 1)};
+`
+export const HeaderDrawerItemNoBorder = styled(Row)`
```

Comment:
You might not need to create another Component just to hide a border. Have you tried to use `:last-child` in `HeaderDrawerItem` to do the same thing?